packages/manager/apps/freefax/src/index.js

```diff
@@ -1,7 +1,9 @@
+import 'bootstrap4';
```

Comment:
you should remove every bootstrap4 js imports, since you imported the utilities function within a sass file

Reply:
You're killing me 😅